dasharo-compatibility/wifi-bluetooth-support.robot

```diff
@@ -30,61 +34,83 @@ WLE001.001 Wireless card detection (Ubuntu 20.04)
     ...    correctly and can be detected from the operating system.
     Skip If    not ${WIRELESS_CARD_SUPPORT}    WLE001.001 not supported
     Skip If    not ${TESTS_IN_UBUNTU_SUPPORT}    WLE001.001 not supported
+    Log To Console    Remember to test all variants of wireless cards.
+    Log    Remember to test all variants of wireless cards.    WARN
     Power On
     Boot System Or From Connected Disk    ubuntu
     Login To Linux
     Switch To Root User
     ${out}=    Execute Command In Terminal    lspci | grep "Network controller:"
     Should Match    ${out}    *${WIFI_CARD_UBUNTU}*
+    Log To Console    The test passed for the ${WIFI_CARD_UBUNTU} wireless card
+    Log    The test passed for the ${WIFI_CARD_UBUNTU} wireless card    WARN
     Exit From Root User
 
 WLE001.002 Wireless card detection (Windows 11)
     [Documentation]    Check whether the Wi-Fi/Bluetooth card is enumerated
     ...    correctly and can be detected from the operating system.
     Skip If    not ${WIRELESS_CARD_SUPPORT}    WLE001.002 not supported
     Skip If    not ${TESTS_IN_WINDOWS_SUPPORT}    WLE001.002 not supported
+    Log To Console    Remember to test all variants of wireless cards.
+    Log    Remember to test all variants of wireless cards.    WARN
     Power On
     Login To Windows
     ${out}=    Execute Command In Terminal    Get-PnpDevice -PresentOnly | Select-String -Pattern "Wi-Fi"
-    Should Contain    ${out}    ${WIFI_CARD}
+    Should Match    ${out}    *${WIFI_CARD_UBUNTU}*
```

Comment:
Shouldn't it be ${WIFI_CARD}?

Reply:
Fixed [here](https://github.com/Dasharo/open-source-firmware-validation/pull/136/files#diff-db99b3a0adebbc880d28cd1f87d49ea87c9b11cd1d3681f599bd1494c26e814cR59)